### ISSUE TITLE
/mailto:/ in a string breaks syntax highlighting

### ISSUE THREAD

**Issue by neerolyte** [atom/language-php#219]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Writing a string with `/mailto:/` breaks syntax highlighting. I first noticed this with a `preg_replace` like in the screen shot below, but I can also reproduce with just `$foo = '/mailto:/';` or `$foo = "/mailto:/";`.

### Steps to Reproduce

1. Create a php file.

2. set its contents to:

```php
<?php
$foo = 'bar';
$match = preg_replace('/mailto:/i', '', $match);
$foo = 'bar';
```

**Expected behavior:** [What you expect to happen]

Syntax highlighting to leave the `string.regexp.single-quoted.php` mode at the end of the regex.

More like this (note that it doesn't break here because I'm using `%` as the delimiters instead of `/`):

![image](https://user-images.githubusercontent.com/1070702/27203488-72482512-5269-11e7-9305-b30bcdbac74d.png)

**Actual behaviour:** [What actually happens]

Rest of code remains in `string.regexp.single-quoted.php` mode.

![image](https://user-images.githubusercontent.com/1070702/27203454-408cc028-5269-11e7-842b-a8304dbbe7af.png)

**Reproduces how often:** [What percentage of the time does it reproduce?]

100%

### Versions

```
$ atom --version
Atom    : 1.17.2
Electron: 1.3.15
Chrome  : 52.0.2743.82
Node    : 6.5.0
$ apm --version
apm  1.17.0
npm  3.10.5
node 6.9.5 x64
python 2.7.12
git 2.12.2
```

OS X 10.12.5

**Comment by 50Wliu** [comment]
Regex.  Always fun trying to handle regex using regex.

**Comment by 50Wliu** [comment]
Turns out this was an issue with language-hyperlink matching trying to match links in regex strings.  Fixed and should be out when Atom 1.20.0 is released.

**Comment by neerolyte** [comment]
Thanks for the update :)
